suite-metrics/ComputeMetrics.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+rem parameters:
+rem %1: SCADE installation directory, e.g. C:\Program Files\ANSYS Inc\v242\SCADE
+rem %2: project
+rem %3: configuration
+
+:: check if there are 3 parameters
```

Comment:
Most actions uses same parameters and checks,. Could it be good to put this code in a separate batch ?
It will only save 20 lines, but make easier the reading and maintenance.